grammars/javascript.cson

```diff
@@ -281,240 +281,283 @@
         'name': 'variable.language.js'
   }
   {
-    'captures':
```

Comment:
`support.class` -> `variable.other.object` for `Sound` as it isn't a built-in class